src/Emitter.ts

```diff
@@ -192,17 +217,17 @@ export class Emitter
 
                     if (!isParentEnum)
                     {
-                        if (interfaceMerge)
-                            parent.children.push(interfaceMerge);
+                        if (interfaceMerge) parent.children.push(interfaceMerge);
+                        if (namespaceMerge) parent.children.push(namespaceMerge);
 
                         parent.children.push(obj);
                     }
                 }
             }
             else
             {
-                if (interfaceMerge)
-                    this._treeRoots.push(interfaceMerge);
+                if (interfaceMerge) this._treeRoots.push(interfaceMerge);
+                if (namespaceMerge) this._treeRoots.push(namespaceMerge);
```

Comment:
Please split these lines a bit, I don't like having code on the same line as an `if`.

Reply:
Done.